jboss/container/java/jvm/bash/artifacts/opt/jboss/container/java/jvm/java-default-options

```diff
@@ -144,6 +145,7 @@ gc_config() {
        "-XX:MaxHeapFreeRatio=${maxHeapFreeRatio} "\
        "-XX:GCTimeRatio=${timeRatio} "\
        "-XX:AdaptiveSizePolicyWeight=${adaptiveSizePolicyWeight} "\
+       "-XX:MetaspaceSize=${metaSpaceSize}m"
```

Comment:
It looks like there is a missing space and "\\ at on this line at the end of this line.

Thi change throws `/opt/run-java/java-default-options: line 149: -XX:MaxMetaspaceSize=192m: command not found` error in the console